### DISCUSSION TITLE
I can't see the process and temp size.

### DISCUSSION THREAD

**Discussion by karasgoon** [swar/Swar-Chia-Plot-Manager#693] [Q&A]
After running python manager.py view, the progress and temp size are not displayed. The Chia mainnet version is 1.1.6. Is it normal?
![aaa](https://user-images.githubusercontent.com/84580793/119151915-c9deb100-ba8a-11eb-879a-3da2c0ac2a5c.JPG)

**Comment by scevaCN** [comment]
me too
